docs/guides/CustomStartupGuide.md

```diff
@@ -84,7 +174,7 @@ Install the media utility tool:
    ```
 
 
-### Get the Device USB Camera Source Code (if not already downloaded)
+### Get the Device USB Camera Source Code
```

Comment:
For this section, I would put the optional configurations first. In the ONVIF guide, the configurations come before the device service is run for the first time, and then any more information is included in the optional section at the end. If there is not enough additional information, then the optional section might be unnecessary.